### DISCUSSION TITLE
queue running

### DISCUSSION THREAD

**Discussion by JulieVelghe** [verbb/formie#1587] [Q&A]
I was wondering,
does the queue ONLY run when someone logs into the control panel?
I have a site where submissions from a formie form get send to a google spreadsheet via a 'miscellaneous' integration. These only seem to get send when I log into the control panel. Or if the queue does run, I haven't waited long enough.
Some sites have integrations with captcha, email marketing etc. and those seem to run fine after the site is set up.
Or do these work differently then the 'miscellaneous' ones?
The current site I'm working on is very much on point, so perhaps only once a week there is a login. It wouldn't be productive to only have that google sheet updated once a week.

**Comment by engram-design** [comment]
This is the default behaviour with Craft, yes. However, there are several methods you can use to change this behaviour. This is detailed in the [docs](https://verbb.io/craft-plugins/formie/docs/get-started/troubleshooting).
